perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -77,6 +77,7 @@ struct objectMessage
 	std::vector<geometry_msgs::Point32> strikesFrame;
 	std::vector<uint32_t> intensityPersist;
 	std::vector<uint32_t> intensityFrame;
+	std_msgs::Header header;
```

Comment:
Looks like good changes to me, but be sure to rebase with dev which contains Ira's latest PR and may already have some of these changes. Looks like you're going to get a not so fun rebase conflict in these files. Up to you to try to actually line by line resolve it or just use Ira's version.